server/src/opldb/fields/federation.rs

```diff
@@ -142,6 +142,8 @@ pub enum Federation {
     #[strum(to_string = "SPF", serialize = "spf")]
     SPF,
     #[strum(to_string = "SVNL", serialize = "svnl")]
+    SSF,
+    #[strum(to_string = "SSF", serialize = "ssf")]    
```

Comment:
The `#[strum]` decorator should go above the line being described. (SVNL needs to be moved back down.)